vaadin-maps-leaflet-flow/src/main/java/software/xdev/vaadin/maps/leaflet/flow/data/LPoint.java

```diff
@@ -26,18 +26,28 @@
 	
 	public LPoint(final double lat, final double lon)
 	{
-		this.coords.add(lat);
-		this.coords.add(lon);
+		coords.add(lat);
```

Comment:
"this." should be added automatically through intellij. Please delete this change.